### PR TITLE
added example

### PR DESCRIPTION
**Pull request by nitin42**
| Q                        | A 
| ------------------------ | ---
| Patch: Bug Fix?          | 
| Major: Breaking Change?  | 
| Minor: New Feature?      | 
| Deprecations?            | 
| Spec Compliancy?         | 
| Tests Added/Pass?        | 
| Fixed Tickets            | `Fixes #1, Fixes #2` 
| License                  | MIT
| Doc PR                   | 
| Dependency Changes       | 

added an example for `babel-plugin-transform-es2015-sticky-regex`.

### CODE REVIEW

File: packages/babel-plugin-transform-es2015-sticky-regex/README.md [hunk 1]
```diff
@@ -2,6 +2,20 @@
 
 > Compile ES2015 sticky regex to an ES5 RegExp constructor
 
+## Example
+
+**In**
+
+```javascript
+/ab+c/i;
```
Comment (existentialism) [inline packages/babel-plugin-transform-es2015-sticky-regex/README.md]: Have you tried running this through this transform? :)

Comment (nitin42) [inline packages/babel-plugin-transform-es2015-sticky-regex/README.md]: Ya I tried with constructor. Removed and added a new example. Tried this !

**Comment by xtuc** [comment]
I don't think it's necessary to get the code coverage to pass since it's only a doc PR.

**Comment by existentialism** [comment]
Closing in favor of #5393

**Pull request closed without merging**